addons/delivery/models/delivery_carrier.py

```diff
@@ -199,7 +200,7 @@ def rate_shipment(self, order):
                 product_currency=company.currency_id
             )
             # apply margin on computed price
-            res['price'] = float(res['price']) * (1.0 + (self.margin / 100.0))
+            res['price'] = float((res['price']) * (1.0 + (self.margin / 100.0))) + self.fixed_margin
```

Comment:
Not sure to understand why you changed the `float`'s parenthesis ?
The float is here only to be sure `res[ 'price'] ` is a float here I think